pkg/config/piped.go

```diff
@@ -116,6 +120,33 @@ func (s *PipedSpec) Validate() error {
 	return nil
 }
 
+// Mask masks confidential fields.
+func (s *PipedSpec) Mask() {
+	if len(s.PipedKeyFile) != 0 {
+		s.PipedKeyFile = maskString
+	}
+	if len(s.PipedKeyData) != 0 {
+		s.PipedKeyData = maskString
+	}
+	s.Git.Mask()
+	for i := 0; i < len(s.ChartRepositories); i++ {
+		s.ChartRepositories[i].Mask()
+	}
+	for i := 0; i < len(s.ChartRegistries); i++ {
+		s.ChartRegistries[i].Mask()
+	}
+	for _, p := range s.CloudProviders {
+		p.Mask()
+	}
+	for _, p := range s.AnalysisProviders {
+		p.Mask()
+	}
+	s.Notifications.Mask()
+	if s.SecretManagement != nil {
+		s.SecretManagement.Mask()
+	}
+}
```

Comment:
I wish a function could iterate all fields to execute the `Mask` function on them.
But I understand it can be done like this for now.